cmd/minikube/cmd/cache.go

```diff
@@ -70,6 +70,19 @@ var deleteCacheCmd = &cobra.Command{
 	},
 }
 
+// reloadCacheCmd represents the cache reload command
+var reloadCacheCmd = &cobra.Command{
+	Use:   "reload",
+	Short: "reload cached images.",
+	Long:  "reloads the cached images specificed by user in the config file.",
```

Comment:
spelling: 'specified'

I'm also not sure this comment is helpful: the user does not know which config file this refers to. Perhaps saying something like "reloads images previously added using the 'add' subcommand"?

Reply:
done